app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/components/ManageFormAccessDialog/ManageFormAccessDialogContext.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useState, ReactNode } from "react";
+
+interface ManageFormAccessDialogContextProps {
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+  formId: string;
+  setFormId: (formId: string) => void;
+  emailList: string[];
+  setEmailList: (emailList: string[]) => void;
+}
+
+export const ManageFormAccessDialogContext = createContext<
+  ManageFormAccessDialogContextProps | undefined
+>(undefined);
+
+export const ManageFormAccessDialogProvider = ({ children }: { children: ReactNode }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [formId, setFormId] = useState("");
+  const [emailList, setEmailList] = useState<string[]>([]);
+
+  return (
+    <ManageFormAccessDialogContext.Provider
+      value={{
+        isOpen,
+        setIsOpen,
+        formId,
+        setFormId,
+        emailList,
+        setEmailList,
+      }}
+    >
+      {children}
+    </ManageFormAccessDialogContext.Provider>
+  );
+};
```

Comment:
Add the  useManageFormAccess here

Reply:
https://github.com/cds-snc/platform-forms-client/pull/4392/commits/fdc4331b68ec4221b12223cbe8f0ad5a4171d3c8